infrastructure/src/main/java/org/corfudb/infrastructure/log/SegmentHandle.java

```diff
@@ -26,10 +26,10 @@
     final long segment;
 
     @NonNull
-    final FileChannel writeChannel;
+    final FileChannelPerf writeChannel;
```

Comment:
When you extend the FileChannelImpl, then you wont need to change these types, which I think we shouldn't do.